frontend/src/pages/modelServing/screens/projects/ModelServingPlatform.tsx

```diff
@@ -127,11 +130,32 @@ const ModelServingPlatform: React.FC = () => {
         )}
       </DetailsSection>
       <ManageServingRuntimeModal
-        isOpen={isOpen}
+        isOpen={platformSelected === ServingRuntimePlatform.MULTI}
         currentProject={currentProject}
-        servingRuntimeTemplates={templatesEnabled}
+        servingRuntimeTemplates={templatesEnabled.filter((template) =>
+          getTemplateEnabledForPlatform(template, ServingRuntimePlatform.MULTI),
+        )}
+        onClose={(submit: boolean) => {
```

Comment:
This close function could be extracted above because it will be reused in both modals.

Reply:
Ah I mean just simply create a function
```
const onModalClose = (submit: boolean) => {
  setPlatformSelected(undefined);
  if (submit) {
    refreshAllProjects();
    refreshServingRuntime();
    refreshInferenceServices();
    setTimeout(refreshTokens, 500); // need a timeout to wait for tokens creation
  }
}
```
and apply it in both modal `onClose` prop `onClose={onModalClose}`